doc/release/1.13.0-notes.rst

```diff
@@ -92,6 +92,10 @@ invokes``ndarray.__getslice__`` (e.g. through ``super(...).__getslice__``) will
 now issue a DeprecationWarning - ``.__getitem__(slice(start, end))`` should be
 used instead.
 
+``PyArray_Descr.elsize`` is now ``-1`` for unsized flexible dtypes
+------------------------------------------------------------------
+Previously it was ``0`` - but that made it impossible to distinguish unsized
+from sized-to-0.
```

Comment:
I was thinking code that assumed setting `elsize` to `0` would allow it to resize would be surprise when it no longer does